recipe/meta.yaml

```diff
@@ -18,21 +18,24 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
-    - dtoolcore
+    - python >=3.6
+    - dtoolcore  # ==3.12.0
     - dtool_cli
-    - dtool_create
-    - dtool_info
+    - dtool_create  # ==0.22.0
+    - dtool_info  # ==0.15.0
     - dtool_symlink
-    - dtool_http
+    - dtool_http  # ==0.3.0
     - dtool_config
 
 test:
   commands:
     - dtool --help
+    # - pip check
```

Comment:
You should add a comment here "why".

Reply:
I try to fix it with in the next 24 hours - but I can add comments if you want.